src/azure-cli/azure/cli/command_modules/resource/custom.py

```diff
@@ -383,7 +386,16 @@ def deploy_arm_template_at_subscription_scope(cmd,
                                               template_file=None, template_uri=None, parameters=None,
                                               deployment_name=None, deployment_location=None,
                                               no_wait=False, handle_extended_json_format=None,
+                                              confirm_with_what_if=None, what_if_result_format=None,
                                               no_prompt=False):
```

Comment:
It is suggested that the newly added parameters should be placed at the end

Reply:
That's good to know. I moved them to the end.